Description: add local libdir to link with grpc
 d/rules set TOPDIR to installed library directory.
Forwarded: not-needed
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2024-08-11

---

Index: grpc/src/ruby/ext/grpc/extconf.rb
===================================================================
--- grpc.orig/src/ruby/ext/grpc/extconf.rb
+++ grpc/src/ruby/ext/grpc/extconf.rb
@@ -120,6 +120,8 @@ env_append 'CPPFLAGS', '-DGRPC_ENABLE_FO
 output_dir = File.expand_path(RbConfig::CONFIG['topdir'])
 grpc_lib_dir = File.join(output_dir, 'libs', grpc_config)
 ENV['BUILDDIR'] = output_dir
+topdir = ENV['TOPDIR'] || '.'
+$LDFLAGS << ' -L' + topdir
 
 strip_tool = RbConfig::CONFIG['STRIP']
 strip_tool += ' -x' if apple_toolchain
